src/Components/FoodForm.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+const initialState = {
+  name: '',
+  calories: 0,
+  image: '',
+  quantity: 0,
```

Comment:
In Foodbox, a quantity of 1 is set to the state.
Will setting a quantity here and passing it to foodBox have an effect?